hnn_core/cell.py

```diff
@@ -148,6 +148,51 @@ def gid(self, gid):
             raise RuntimeError('Global ID for this cell already assigned!')
 
 
+class Section:
+    """Section class.
+
+    Parameters
+    ----------
+    L : float
+        length of a section in microns
+    diam : float
+        diameter of a section in microns
+    cm : float
+        membrane capacitance in micro-Farads
+    Ra : float
+        axial resistivity in ohm-cm
+    """
+    def __init__(self, L, diam, Ra, cm, sec_pts=None):
+
+        self._L = L
+        self.diam = diam
+        self.Ra = Ra
+        self.cm = cm
+        if sec_pts is None:
+            sec_pts = list()
+        self.sec_pts = sec_pts
+
+        self.mechs = dict()
+        self.syns = list()
+
+    @property
+    def L(self):
+        return self._L
+
+    @L.setter
```

Comment:
What sort of checks do you envision if `L` is user-settable? I'm thinking #342 again here...

Reply:
For starters, we could throw a warning if a pyramidal cell radius exceeds 1/2 of `_inplane_distance`?